### PR TITLE
Implement the ability to view any account.

### PR DESCRIPTION
**Pull request by clindsey4**
Implemented the ability for officer & advisors to view the account page of any user.

### CODE REVIEW

File: src/app/[lang]/account/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+) {
+
+    // get the current session
+    const session = await getActiveSession(cookies())
+
+    // redirect if the user is not able to view any account
+    if (!session || viewAnyAccountMinAccessLevel > session.user.accessLevel) return redirect("./")
+
+    // parse the user email
+    const user_email = `${params.id}@${process.env.GOOGLE_OAUTH_HD || ''}`
```
Comment (djimenezmsu) [inline src/app/[lang]/account/[id]/page.tsx:31]: The session doesn't contain the user email? Weird that google wouldn't provide that

File: src/data/initializers/webData.ts [hunk 2]
```diff
@@ -14,6 +14,21 @@ export default function init(
...
+    try {
+        if (exists) {
+            database.prepare(`UPDATE users
+            SET access_level = 2 
+            WHERE email = 'clindsey4@murraystate.edu'`).run()
+            database.prepare(`UPDATE users 
+            SET access_level = 2 
+            WHERE email = 'awatkins23@murraystate.edu'`).run()
+        } else {
+            database.prepare(`INSERT INTO users (email, given_name, family_name, picture, access_level) VALUES ("clindsey4@murraystate.edu", "", "", "", 2)`).run()
```
Comment (djimenezmsu) [inline src/data/initializers/webData.ts:27]: Why not fill in your given name, family name? Also, we should create a developer account to use with unrestricted access. For QA/Testing purposes

**Review by djimenezmsu — APPROVED** [review]
Very Big Brother :tm: to be able to view users like this haha

File: src/app/[lang]/account/[id]/page.tsx (hunk 1 above)
Comment (clindsey4) [reply src/app/[lang]/account/[id]/page.tsx:31]: It does, it's just that only emails from murraystate.edu are allowed to create accounts/sign in, so there's no point in storing it in a session.

File: src/data/initializers/webData.ts (hunk 2 above)
Comment (clindsey4) [reply src/data/initializers/webData.ts:27]: Yeah, this was temporary while I was struggling to host the website yesterday. I'll add something more secure & less hardcoded in the future.

**Pull request merged**